### ISSUE TITLE
Problems with folding //#region area

### ISSUE THREAD

**Issue by Jusys** [Microsoft/vscode#47309]
- VSCode Version: 1.22.1
- OS Version: Windows 7 32bit

Folding symbol next to the //#region and folding of //#region area behaves strangely in v1.22.1. 

![clipboard04](https://user-images.githubusercontent.com/30110941/38417904-d7b1831e-39a2-11e8-91bd-69c0954d8eaa.jpg)
![clipboard06](https://user-images.githubusercontent.com/30110941/38417907-db6fca56-39a2-11e8-891c-2359eaf9a5c6.jpg)

Steps to Reproduce:
1. JavaScript, enter this code in one window and another and try to switch between windows (ctrl+tab):

```javascript
const ddd = '';

module.exports = {
	blablabla: aaa,
	blablabla2: bbb,
}

//#region function aaa()
/**
 * 
 */
function aaa()
{

}
//#endregion

//#region function bbb()
/**
 * 
 * @param param 
 */
function bbb(param)
{

}
//#endregion
```

Does this issue occur when all extensions are disabled?: I have no extensions installed

Sorry for my English.

**Comment by Jusys** [comment]
P.S. sorry for duplicate - it took some time for me to write code, capture screens, comment them and try to find what's causing this bug.

**Comment by kash4kev** [comment]
Fixed on my install of 1.22.1 by adding:

**"typescript.experimental.syntaxFolding": false** to my workspace **settings.json**.

**Comment by mjbvz** [comment]
Duplicate of #44530 

Setting `"typescript.experimental.syntaxFolding": false` will disable the new syntax based folding and should fix this issue.
